src/core/layout/qgslayoutaligner.cpp

```diff
@@ -302,13 +367,17 @@ QString QgsLayoutAligner::undoText( Distribution distribution )
     case DistributeLeft:
       return QObject::tr( "Distribute Items by Left" );
     case DistributeHCenter:
-      return QObject::tr( "Distribute Items by Center" );
+      return QObject::tr( "Distribute Items by Horizontal Center" );
+    case DistributeHSpace:
+      return QObject::tr( "Distribute Items by Horizontal Equispaced" );
```

Comment:
```suggestion
      return QObject::tr( "Distribute Items Horizontally Equispaced" );
```

or similar sounds better to me. Anyone with native English skills around? @NathanW2 @nyalldawson

Reply:
It should be "Distribute Horizontal Spacing Equally"/...

Reply:
The reason was because the rest of the items of that menu follow the same "Distribute Items by" "pattern", I will follow your suggestion about it.

Reply:
What about .ui file below ?
```xml
src/ui/layout/qgslayoutdesignerbase.ui:    <string>Distribute &amp;Horizontally Equispaced</string>
src/ui/layout/qgslayoutdesignerbase.ui:    <string>Distribute Vertically &amp;Equispaced</string>
```